core/benches/blocks/apply_blocks_oneshot.rs

```diff
@@ -9,10 +9,16 @@ mod apply_blocks;
 
 use apply_blocks::WsvApplyBlocks;
 
-#[tokio::main]
-async fn main() {
-    iroha_logger::test_logger();
+fn main() {
+    let rt = tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
+        .build()
+        .expect("Failed building the Runtime");
+    {
+        let _guard = rt.enter();
+        iroha_logger::test_logger();
+    }
```

Comment:
Logger needs runtime context...?

Reply:
yes, initialization of logger, `LoggerHandle::new` calls `tokio::spawn`